src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -314,13 +365,13 @@ private static boolean hasJavadocCommentOnToken(DetailAST methodDef, int tokenTy
      * @param token tree token.
      * @return true if a javadoc comment exists under the token.
      */
-    private static boolean branchContainsJavadocComment(DetailAST token) {
+    private boolean branchContainsJavadocComment(DetailAST token) {
```

Comment:
if you make `requiredJavadocPhrase` static, can we keep methods to stay static ?

Reply:
Seems like Checker does not like static properties:
![image](https://user-images.githubusercontent.com/31252532/105098168-aa516080-5a77-11eb-9615-bc89bd2e5341.png)

Reply:
@nmancus1 , please try to share text form of error all the time, image is last resort.

Reply:
this error means you have problem in config with unrecognized name of property

Reply:
sorry, I now got a problem, my bad, property of Check can not be static. I recall review item.